framework/include/utils/MathUtils.h

```diff
@@ -15,6 +15,9 @@
 #include "libmesh/utility.h"
 #include "libmesh/numeric_vector.h"
 #include "libmesh/compare_types.h"
+#include "libmesh/point.h"
+
+using libMesh::Point;
```

Comment:
this will end up active on every source file that includes this header right?
I think for this reason we ll want to avoid doing that

Reply:
`using` can be scoped to the function it is needed in.

Reply:
Put the using declaration into each function that needs it.

Reply:
dont know.
MathUtils shouldnt modify the way people deal with a geometrical object

Reply:
Just using Point caused tests in Non unity build to fail with incomplete return type error

Reply:
> MathUtils shouldnt modify the way people deal with a geometrical object

Agree, but that horse is out of the barn anyways, just FYI. I do think that putting the declaration in the function scope is the right thing to do, either way. No need to at yet another using to the global scope.